blueocean-web/package.json

```diff
@@ -30,31 +28,41 @@
     "giti": "1.1.3",
     "gulp": "3.9.1",
     "jquery-detached": "2.1.4-v4",
-    "moment": "2.15.1",
     "ncp": "2.0.0",
-    "react-router": "2.8.1",
     "reactify": "1.1.1",
     "zombie": "4.2.1"
   },
   "dependencies": {
+    "@jenkins-cd/blueocean-core-js": "0.0.170",
+    "@jenkins-cd/design-language": "0.0.158",
```

Comment:
So core-js and JDL are now listed in deps?  I assume they are still being supplied by maven-blueocean-plugin? Can you explain this change?

Reply:
Yes, these are still provided by the `blueocean-maven-plugin` I fixed all the `package.json` to have runtime dependencies under `dependencies` and build-time under `devDependencies`, I think this is why the downstream `moment-duration-format` issue was happening, because the actual _runtime_ dependencies were wrong

Reply:
You mean that `moment-duration-format` version was wrong, but only in the context of the tests? Interesting. I had thought at the end of the day, browserify didn't really care whether a lib was a `dep` or `devDep` and would `require` whatever was there. Unless npm is doing something interesting like applying different dedup logic between runtime and development dep trees?

Maybe the version numbers we use are totally meaningless but maybe there's some way to make that more obvious, maybe adding some kind of qualifier or even aligning the npm module versions w/ the hpi versions... maybe logging warnings if they don't match? I dunno, my thoughts here aren't fully formed but gut tells me duplicating these is a bummer.

Reply:
Right, duplicating these are indeed a bummer, maybe we can just set it and forget it at something blatantly obvious like `0.0.0-provided`